SMLHelper/Utility/ImageUtils.cs

```diff
@@ -3,6 +3,11 @@
     using System.IO;
     using UnityEngine;
     using Logger = V2.Logger;
+#if SUBNAUTICA
+    using Sprite = Atlas.Sprite;
+#elif BELOWZERO
+    using Sprite = UnityEngine.Sprite;
+#endif
```

Comment:
As I said a while ago in the Discord, we should encourage modders to use `UnityEngine.Sprite` for both games, not `Atlas.Sprite` for Subnautica and `UnityEngine.Sprite` for Below Zero.

Reply:
As I also explained, that is not going to be easy to accommodate in classes like these where the sprite type is only present in the return value.

Also, `Atlas.Sprite` is still very much in use in Subnautica, with most of the UI elements in the game code using the `Atlas` type instead of the `UnityEngine` type.

Reply:
Then I guess we shall make the appropriate changes if/when the `Atlas.Sprite` removal comes to Subnautica